src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -40,26 +83,28 @@ module.exports = kind({
 	pattern: 'activity',
 	classes: 'moon enyo-fit enyo-unselectable',
 	components: [
-		{kind: Panel, classes: 'moon-6h', title: 'Menu', components: [
-			{kind: Item, content: 'Scroll'},
-			{kind: Item, content: 'the'},
-			{kind: Item, content: 'Data Grid List'},
-			{kind: Item, content: 'to'},
-			{kind: Item, content: 'the'},
-			{kind: Item, content: 'Right!'}
-		]},
-		{kind: Panel, joinToPrev: true, title: 'Data Grid List', headerComponents: [
-			{kind: ToggleButton, content: 'Selection', name: 'selectionToggle'},
+		{kind: Panel, name: 'listPanel', title: 'Data Grid List', headerComponents: [
+			{kind: ToggleButton, content: 'Selection', name: 'selectionToggle', onChange: 'selectionChanged'},
 			{kind: ContextualPopupDecorator, components: [
 				{kind: ContextualPopupButton, content: 'Selection Type'},
 				{kind: ContextualPopup, classes: 'moon-4h', components: [
-					{kind: RadioItemGroup, name: 'selectionTypeGroup', components: [
+					{kind: RadioItemGroup, name: 'selectionTypeGroup', onActiveChanged: 'selectionTypeChanged', components: [
 						{content: 'Single', value: 'single', selected: true},
 						{content: 'Multiple', value: 'multi'},
 						{content: 'Group', value: 'group'}
 					]}
 				]}
 			]},
+			{kind: ContextualPopupDecorator, components: [
+				{kind: ContextualPopupButton, content:'Item Type'},
+				{kind: ContextualPopup, classes:'moon-6h', components: [
+					{kind: RadioItemGroup, name: 'itemTypeGroup', onActiveChanged: 'itemTypeChanged', components: [
```

Comment:
The `onActiveChanged` event is being fired when this popup is first opened, causing an undesired regeneration of the DataGridList. Seems like this is a pre-existing behavior with `moon.SelectableItem`, as `updateActiveValue` is called in `rendered`, which doesn't occur until the popup is actually opened (rendering the item) in this case.

@germboy (because of your history making some related tweaks in https://github.com/enyojs/moonstone/commit/77f8c73fd1c06b10fe57ed6b257b2f736c53d668) @ryanjduffy any thoughts on this and/or are there issues you can think of if we instead call `selectedChanged`(instead of `updateSelectedValue`) in `create` and remove the call in `rendered`? I realize that this can definitely be outside the scope of this PR, but if we wanted to leave the existing behavior of `moon.SelectableItem`, I think we should probably make a tweak to the sample to prevent the observed behavior.